README.md

```diff
@@ -90,7 +90,7 @@ For an overview of the full workflow, see the [workflow diagram](./docs/workflow
 
 - **🍎 Apple M1/M2/M3 Mac or 🐧 Linux system** (tested on Fedora). We anticipate support for more operating systems in the future.
 
-   📋  When installing the `ilab` CLI on macOS, you may have to run the `xcode-select --install` command, installing the required packages listed.
+   📋  When installing the `ilab` Core on macOS, you may have to run the `xcode-select --install` command, installing the required packages listed.
```

Comment:
```suggestion
   📋  When installing InstructLab Core on macOS, you may have to run the `xcode-select --install` command, installing the required packages listed.
```

Reply:
I'd remove "the"

Reply:
updated `InstructLab Core` without `the`